code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -56,14 +75,15 @@
 			target << "<span class='userdanger'>Your gaze is forcibly drawn into [usr]'s eyes, and you are mesmerized by the heavenly lights...</span>"
 		else
 			target << "<span class='userdanger'>Red lights suddenly dance in your vision, and you are mesmerized by their heavenly beauty...</span>"
-		target.Stun(3) //Roughly 30% as long as the normal one
-		M.silent += 3
+		target.Stun(5) //Roughly 50% as long as the normal one
```

Comment:
This actually corresponds to 4 extra seconds, since each point of stun takes 2 seconds to deplete.